ClientApp/viking-vault/src/RegisterForm.tsx

```diff
@@ -0,0 +1,233 @@
+import React from "react";
+import {ResponseModal} from "./ReponseModal";
+import {Redirect, BrowserRouter as Router} from "react-router-dom";
+
+const url = "https://localhost:44323/api/";
```

Comment:
Since this is used in other areas of the code, its better to make a file of constants and use the same constant all throughout the application, it prevents a lot of bugs and when you deploy it in za cloud you will need to change the api endpoint everywhere.